.github/workflows/windows.yml

```diff
@@ -0,0 +1,45 @@
+name: arviz-compatibility
```

Comment:
Forgot to change the name?

Reply:
probably, but this one's also testing arviz compatibility:
```
          conda remove arviz -y
          pip install git+git://github.com/arviz-devs/arviz.git
```
maybe we can remove that and just have it as a suite of windows tests (which I presume we'll want to expand once we have these in)